plugins/processors/noise/README.md

```diff
@@ -0,0 +1,60 @@
+# Noise Processor
+
+The *Noise* processor is used to add noise to numerical field values. For each field a noise is generated using a defined probability densitiy function and added to the value. The function type can be configured as _Laplace_, _Gaussian_ or _Uniform_.
+Depending on the function, various parameters need to be configured:
+
+### Laplace:
+* `noise_type = laplacian`
+* `scale`: also referred to as _diversity_ parameter, regulates the width & height of the function, a bigger `scale` value means a higher probability of larger noise, default set to 1.0
+* `mu`: location of the curve, default set to 0.0
+
+### Gaussian:
+* `noise_type = gaussian`
+* `mu`: mean value, default set to 0.0
+* `scale`: standard deviation, default set to 1.0
+
+### Uniform
+* `noise_type = uniform`
+* `min`: minimal interval value, default set to -1.0
+* `max`: maximal interval value, default set to 1.0
+
+### Configuration
+Depending on the choice of the distribution function, the respective parameters must be set. Default settings are `noise_type = "laplacian"` with `mu = 0.0` and `scale = 1.0`:
+
+```toml
+[[processors.noise]]
+  scale = 1.0
+  mu = 0.0
+  noise_type = "laplacian"
+  include_fields = []
+  exclude_fields = []
```

Comment:
Maybe leave out those in the example or use them? It might confuse people as often they just copy those configs for a start.